pkg/plugin/infiniband/infiniband_linux.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Package infiniband contains the Retina infiniband plugin. It gathers infiniband statistics and debug status parameters.
+package infiniband
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	hubblev1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	kcfg "github.com/microsoft/retina/pkg/config"
+	"github.com/microsoft/retina/pkg/log"
+	"github.com/microsoft/retina/pkg/plugin/api"
+	"go.uber.org/zap"
+)
+
+// New creates a infiniband plugin.
+func New(cfg *kcfg.Config) api.Plugin {
+	return &infiniband{
+		cfg: cfg,
+		l:   log.Logger().Named(string(Name)),
+	}
+}
+
+func (lu *infiniband) Name() string {
```

Comment:
likely this was copypasted from linuxutils `lu`, but pointer receiver should probably be shortened version of the struct it's referencing 

```suggestion
func (ib *infiniband) Name() string {
```

Reply:
Updated.